spec/appSpec.js

```diff
@@ -78,19 +78,6 @@ describe( "Creating an App", function(){
 		});
 	});
 
-	it( "throws on a missing output element ID", function() {
```

Comment:
When changing the behaviour you shouldn't delete the test case that tested this behaviour, instead you should change the expected behaviour of the test.

In this case the spec has changed from "throws on a missing output element ID" -> "works when no output element ID has been passed".